### PR TITLE
Send all the headers which are correctly prefixed within request

### PR DESCRIPTION
**Pull request by a-urth**
#25

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]

File: trino/trino.go [hunk 1]
```diff
@@ -101,18 +101,19 @@ var (
 const (
 	preparedStatementHeader    = "X-Trino-Prepared-Statement"
```
Comment (hashhar) [inline trino/trino.go:102]:
```suggestion
	preparedStatementHeader    = trinoHeaderPrefix+`Prepared-Statement`
```

**Review by hashhar — APPROVED** [review]
LGTM % comment

File: trino/trino.go [hunk 2]
```diff
@@ -649,7 +650,7 @@ func (st *driverStmt) exec(ctx context.Context, args []driver.NamedValue) (*stmt
 			if err != nil {
 				return nil, err
 			}
-			if arg.Name == trinoUserHeader {
+			if strings.HasPrefix(arg.Name, trinoHeaderPrefix) {
```
Comment (losipiuk) [inline trino/trino.go:653]: It does not look right. We are still adding `trinoUserHeader` always below.

Comment (a-urth) [reply trino/trino.go:653]: very good catch, thanks @losipiuk

**Review by losipiuk — APPROVED** [review]

**Comment by losipiuk** [comment]
@a-urth can you please squash the commits?

**Comment by a-urth** [comment]
@losipiuk squashed commits into one and changed message accordingly

**Pull request merged**